packages/autocomplete/src/examples/multiselect.md

```diff
@@ -0,0 +1,304 @@
+        );
+      }
+
+      return labels;
+    };
+
+    return (
+      <FieldContainer>
+        {({ getLabelProps, getInputProps: getFieldInputProps }) => (
+          <TextGroup style={{ maxWidth: 500, minHeight: 300 }}>
+            <Label {...getLabelProps()}>Advanced Multi-Select Example</Label>
+            <AutocompleteContainer
+              isOpen={state.isOpen}
+              tagFocusedKey={state.tagFocusedKey}
+              onSelect={selectedKey => {
+                let natoPhonetics = state.natoPhonetics;
+                const selectedKeys = Object.assign({}, state.selectedKeys);
+
+                if (selectedKeys[selectedKey]) {
+                  delete selectedKeys[selectedKey];
+                } else {
+                  selectedKeys[selectedKey] = true;
+                }
+
+                setState({ selectedKeys, natoPhonetics, inputValue: '' });
+
+                return true;
+              }}
+              onStateChange={newState => {
+                let inputValue = state.inputValue;
+
+                if (typeof newState.isOpen !== 'undefined' && !newState.isOpen) {
+                  newState.inputValue = '';
+                }
+
+                setState(newState);
+              }}
+              trigger={({
+                getTriggerProps,
+                getInputProps,
+                getTagProps,
+                triggerRef,
+                inputRef,
+                isOpen,
+                tagFocusedKey
+              }) => {
+                return (
+                  <FauxInput
+                    {...getTriggerProps({
+                      open: isOpen,
+                      small: true,
+                      select: true,
+                      tagLayout: true,
+                      focused: isOpen || typeof tagFocusedKey !== 'undefined' || state.isFocused,
+                      inputRef: ref => {
+                        this.wrapperRef = ref;
+                        triggerRef(ref);
+                      }
+                    })}
+                  >
+                    {renderTags(
+                      isOpen || typeof tagFocusedKey !== 'undefined' || state.isFocused,
+                      state.selectedKeys,
+                      tagFocusedKey,
+                      getTagProps
+                    )}
+
+                    <Input
+                      {...getInputProps(
+                        getFieldInputProps({
+                          bare: true,
+                          innerRef: inputRef,
+                          value: state.inputValue,
+                          onChange: e => {
+                            setState({ inputValue: e.target.value });
+                          },
+                          onKeyDown: e => {
+                            if (tagFocusedKey !== undefined) {
+                              if (
+                                e.keyCode === KEY_CODES.DELETE ||
+                                e.keyCode === KEY_CODES.BACKSPACE
```

Comment:
The backspace to focus + another backspace to delete is tedious. We should be deleting on backspace as we did previously. An implementation of `cmd-Z` would be required for convenient undo of an unintended action.

Or am I missing a hard a11y requirement here?

Reply:
I guess the trouble there might be conveying what happened to a screen read if it never focuses the tag and just delete it

Reply:
A problem mitigated somewhat by cursoring-to the tag before hitting delete.

Reply:
> A problem mitigated somewhat by cursoring-to the tag before hitting delete

That was the original intention and is the current behavior in all multi-select/tag collections in Support currently.

A deletion is an action performed against an item and traditionally you have to be cursored to an item to perform an action.

_Side Notes_: It's important to get the example right for this, but this isn't something that is enforced by the `AutocompleteContainer` logic. So there is flexibility for either way.

Reply:
After sitting down with Allison, we decided to add in the "auto-delete" functionality for the example.  This now more closely matches the interaction in the new Gmail multiselect.

Consumers still have all the customization to go either way, but the example is just a little more opinionated now.